src/core/executors/ExecutorBase.ts

```diff
@@ -85,7 +78,6 @@ export default abstract class ExecutorBase {
   }
 
   protected _flushDeltas(): void {
-    logMethodCall('ExecutorBase._flushDeltas');
```

Comment:
Why remove this log?

Reply:
This log gets called every second and I thought it cluttered the console, but I guess you can filter the logs. I'll add it back

Reply:
ah I see, ya probably better to remove this line then